src/generator/backend/generator.js

```diff
@@ -0,0 +1,258 @@
+'use strict';
+
+var exports = module.exports = {};
+
+const fs = require('fs');
+const moment = require('moment');
+const handlebars = require('handlebars');
+const AdmZip = require('adm-zip');
+
+const tpl = handlebars.compile(fs.readFileSync(__dirname + '/schedule.tpl').toString('utf-8'));
+const sessionsData = require('../../../mockjson/sessions.json');
+const speakersData = require('../../../mockjson/speakers.json');
+const servicesData = require('../../../mockjson/event.json');
+const sponsorsData = require('../../../mockjson/sponsors.json');
+
+if(!String.linkify) {
+  String.prototype.linkify = function() {
+    var urlPattern = /\b(?:https?|ftp):\/\/[a-z0-9-+&@#\/%?=~_|!:,.;]*[a-z0-9-+&@#\/%=~_|]/gim;
+    var pseudoUrlPattern = /(^|[^\/])(www\.[\S]+(\b|$))/gim;
+    var emailAddressPattern = /[\w.]+@[a-zA-Z_-]+?(?:\.[a-zA-Z]{2,6})+/gim;
+    return this
+            .replace(urlPattern, '<a href="$&">$&</a>')
+            .replace(pseudoUrlPattern, '$1<a href="http://$2">$2</a>')
+            .replace(emailAddressPattern, '<a href="mailto:$&">$&</a>');
+  };
+}
+
+handlebars.registerHelper('linkify', function(options) {
+  var content = options.fn(this);
+
+  return new handlebars.SafeString(content.linkify());
+});
+
+function slugify(str) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use the function form of "use strict".](https://www.codacy.com/app/triplez/open-event-webapp/file/2710180114/issues/source?bid=3233857&fileBranchId=3342834#l34)